backend/api/main.py

```
@@ -268,6 +276,46 @@ async def list_sessions():
     }
 
+# Kakao Pay Payment Endpoints
+
+@app.post("/payment/ready", response_model=PaymentReadyResponse)
+async def payment_ready(payment_request: PaymentReadyRequest):
+    """카카오페이 결제 준비 API"""
+    try:
+        response = await kakao_pay_service.prepare_payment(payment_request)
+        return response
+    except PaymentErrorResponse as e:
+        raise HTTPException(status_code=400, detail=e.dict())
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)